src/App.tsx

```diff
@@ -14,7 +14,17 @@ export const App = () => {
       </div>
 
       <div className="sidebar">
-        <FindMovie />
+        <FindMovie
+          onMovieAdd={(movie) => {
+            const haveThisMovieInList = movies.some(movieInList => (
+              movieInList.imdbId === movie.imdbId
+            ));
+
+            if (!haveThisMovieInList) {
+              setMovies([...movies, movie]);
+            }
+          }}
```

Comment:
It is better to avoid such logic in props, you can write a helper function for this and pass it